README.md

```diff
@@ -34,8 +34,8 @@ https://github.com/askui/intent-pilot/assets/106730702/582d7dec-e3ff-43fd-9ab7-0
 - Python 3.9 or later
 - OpenAI Key
 - AskUI token
-    - For starters, we provide a global AskUI access token with 4000 free credits. `ASKUI_WORKSPACE_ID`: 'e26b43ea-a18f-4cd8-a6f1-e1e41dddec18' and `ASKUI_TOKEN`: 'gbwdeDjlWBrsr8nLjOVB'
-    - After that, you can get your own AskUI access token by signing up at [AskUI](https://askui.com)
+    - `ASKUI_WORKSPACE_ID` and `ASKUI_TOKEN` are needed in `.env` file to get the product running.
+    - You can easily get your own AskUI credentials by signing up at [AskUI](https://askui.com)
```

Comment:
Please remove the easily. It can be patronising for users 😃 .

Reply:
Lol, true. I thought it would encourages people to signup. Removing it.